packages/common/src/projects/_internal/ProjectUiSchemaSettings.ts

```diff
@@ -0,0 +1,40 @@
+import { IArcGISContext } from "../../ArcGISContext";
+import { IUiSchema } from "../../core/schemas/types";
+import { IHubProject } from "../../core/types";
+
+/**
+ * @private
+ * constructs the settings uiSchema for Hub Projects.
+ * This defines how the schema should be rendered
+ * in the project settings pane
+ */
+export const buildUiSchema = async (
+  i18nScope: string,
+  options: Partial<IHubProject>,
+  context: IArcGISContext
+): Promise<IUiSchema> => {
+  return {
+    type: "Layout",
+    elements: [
+      {
+        type: "Section",
+        labelKey: `${i18nScope}.sections.mapSettings.label`,
```

Comment:
Like I mentioned on the opendata-ui PR, you can probably move this into the `shared` namespace.  See the `getImageThumbnailUiSchemaElement()` helper for an example.